capstone/client/src/components/StoresPage.js

```diff
@@ -44,12 +46,9 @@ class StorePage extends Component {
           distanceValue : this.props.store.state.distanceValue,
           latitude : this.props.store.state.latitude,
           longitude : this.props.store.state.longitude,
-        }))
-    }
-
-    componentDidMount = () => {
-      // Get Stores from database.
-      this.getStores();
+        }));
+        // Get Stores from database.
+        this.getStores();
```

Comment:
I think async requests should go in `componentDidiMount`, where we can call `setState`. Maybe that's why we have the UI lag on the front page. See
https://medium.com/all-about-react/react-basic-constructor-componentdidmount-and-render-9476f8d28f0f
https://stackoverflow.com/questions/55182526/fetching-data-in-constructor